tests/test_camera.py

```diff
@@ -104,27 +104,25 @@ def test_camera_translate_point_to_game_space(camera, position, point, expected)
     assert camera.translate_point_to_game_space(point) == expected
 
 
-# @pytest.mark.skip(reason="Test for old camera. Will want to restore this functionality in new camera.")
-# def test_sprite_in_viewport():
-#     # Added the expected pixel ratio due to change in default breaking this test.
-#     # 80 is the legacy value.
-#     cam = OldCamera(viewport=(0, 0, 800, 600), pixel_ratio=80)
-#
-#     class Thing(Sprite):
-#         def __init__(self, position=Vector(2, 2)):
-#             super().__init__()
-#             self.size = 2
-#             self.position = position
-#
-#     sprite_in = Thing(Vector(-3, -1))
-#     sprite_half_in = Thing(Vector(5, -2))
-#     sprite_out = Thing(Vector(2, 5))
-#
-#     assert not cam.in_frame(sprite_out)
-#     assert cam.in_frame(sprite_in)
-#     assert cam.in_frame(sprite_half_in)
-#
-#
+@pytest.mark.parametrize("input_position, expected", [
+    [Vector(-3, 1), True],  # Fully inside the camera's view
+    [Vector(5, -2), True],  # partially inside the camera's view
+    [Vector(2, 6), False],  # well outside the Camera's view.
+    [Vector(6, 0), False],  # Outside with edges touching (horizontal)
+    [Vector(0, 4.75), False],  # Outside with edges touching (vertical)
+])
+def test_sprite_in_view(camera, input_position, expected):
+
+    class Thing(Sprite):
```

Comment:
Out of curiosity, any reason for the class instead of assigning at instantiation?

Reply:
>.> Laziness. It was already set up this way, and I didn't think about changing this portion. Will totally change it, though.

Reply:
Naw, i thought about asking for the change, but honestly it doesn't matter and variety in the test suite is good.